src/sentry/receivers/onboarding.py

```diff
@@ -56,9 +58,17 @@ def record_new_project(project, user, **kwargs):
         user_id = default_user_id = user.id
     else:
         user = user_id = None
-        default_user_id = (
-            Organization.objects.get(id=project.organization_id).get_default_owner().id
-        )
+        try:
+            default_user_id = (
+                Organization.objects.get(id=project.organization_id).get_default_owner().id
+            )
+        except IndexError:
+            logging.getLogger("sentry").warn(
```

Comment:
@dcramer Why not use `logging.getLogger(__name__)`? I don't think we want just `sentry` here.

Reply:
do we use sentry.receivers.onboarding as a logger anywhere else? I know sentry is our generic logger name.

Reply:
@dcramer Don't think so but that shouldn't matter. We just want the `name` field in Kibana to match the file path so we can figure out what file generated the log message.

Reply:
Is that a new standard we're following, and if so is it documented/standardized on by all? That never used to be the case. Loggers often get hard to configure if you use filenames, and logging already has a filename attribute that can be sourced with it.

e.g. we have the sentry audit logger which is explicitly named so we can route certain kinds of actions, unrelated to the filename they're in

Reply:
fwiw looks like most loggers are still explicitly named (and not directly connected to the filepath)

Reply:
@dcramer That is true. I am not sure if we have a standard. @wedamija Do you know? Anyways, just having the name `sentry` is not going to be helpful. Something more descriptive would be better.

Reply:
I don't think there's a standard, both explicit names and `__name__` are used randomly.